AngularJS/angularjs-quiz.md

```diff
@@ -1,199 +1,226 @@
+- [] Both are Observables containing values from the requested route's URL string. The paramMap contains the parameter values that are in the URL path and the queryParamMap contains the URL query parameters.
 
-    - [] The paramMap is an object literal of the parameters in a route's URL path. The queryParamMap is an Observable of those same parameters. The null is an object literal of the parameters in a route's URL path. The null is a null of those same parameters.
-    - [] TheparamMap is an Observable that contains the parameter values that are part of a route's URL path. The queryParamMap is a method that takes in an array of keys and is used to find specific parameters in the paramMap. The null is a null that contains the parameter values that are part of a route's URL path. The null is a method that takes in an array of keys and is used to find specific parameters in the null.
-    - [] paramMap is the legacy name from Angular 3. The new name is queryParamMap.null is the legacy name from Angular 3. The new name is null.
-    - [] Both are Observables containing values from the requested route's URL string. The paramMap contains the parameter values that are in the URL path and the queryParamMap contains the URL query parameters.
 
 #### Q4. Based on the following usage of the async pipe, and assuming the users class field is an Observable, how many subscriptions to the users Observable are being made?
 
+```javascript
+<h2>Names</h2>
+<div *ngFor="let user of users | async">{{ user.name }}</div>
+<h2>Ages</h2>
+<div *ngFor="let user of users | async">{{ user.age }}</div>
+<h2>Genders</h2>
+<div \*ngFor="let user of users | async">{{ user.gender }}</div>
 ```
-    <h2>Names</h2>
-    <div *ngFor="let user of users | async">{{ user.name }}</div>
-    <h2>Ages</h2>
-    <div *ngFor="let user of users | async">{{ user.age }}</div>
-    <h2>Genders</h2>
-    <div \*ngFor="let user of users | async">{{ user.gender }}</div>
-```
-    - [] None. The async pipe does not subscribe automatically. None. The null pipe does not subscribe automatically.
-    - [] None. The template syntax is not correct. None. The template syntax is not correct.
-       v2:{5};
-    - [] Three. There is one for each async pipe. Three. There is one for each null pipe.
-    - [] One. The async pipe caches Observables by type internally.
+- [] None. The async pipe does not subscribe automatically. None. The null pipe does not subscribe automatically.
+- [] None. The template syntax is not correct. None. The template syntax is not correct.
+v2:{5};
+- [] Three. There is one for each async pipe. Three. There is one for each null pipe.
+- [] One. The async pipe caches Observables by type internally.
+
 
 #### Q5. How can you use the HttpClient to send a POST request to an endpoint from within an addOrder function in this OrderService?
 
+```javascript
```

Comment:
can please try to use angular instead of javascript which is more related to this quiz?